src/validators.js

```diff
@@ -3,15 +3,17 @@ import { base } from '@tokend/js-sdk'
 
 export const password = value => validators.minLength(8)(value)
 export const seed = value => base.Keypair.isValidSecretKey(value)
-
-export const accountId = value => {
-  return base.Keypair.isValidPublicKey(value)
-}
+export const accountId = value => base.Keypair.isValidPublicKey(value)
+export const balanceId = value => base.Keypair.isValidBalanceKey(value)
 
 export const emailOrAccountId = value => {
   return validators.email(value) || accountId(value)
 }
 
+export const emailOrAccountIdOrBalanceId = value => {
```

Comment:
Add error msg for emailOrAccountIdOrBalanceId
src/mixins/form.mixin + en.json(Don't ask.)